### PR TITLE
SW-3352 Required Field Fixes

### PR DESCRIPTION
**Pull request by charlie-tf**
- Summary of Progress denoted as required
- Checked required fields validation
- denote SDG textfields as required

image
image

### CODE REVIEW

File: src/components/Reports/ReportFormAnnual.tsx [hunk 1]
```diff
@@ -372,28 +372,33 @@ export default function ReportFormAnnual(props: ReportFormAnnualProps): JSX.Elem
...
-                validate &&
-                !report.annualDetails?.sustainableDevelopmentGoals[
-                  report.annualDetails?.sustainableDevelopmentGoals?.findIndex((s) => s.goal === key)
-                ].progress
-                  ? strings.REQUIRED_FIELD
-                  : ''
-              }
-            />
+              />
+              <Typography fontSize='14px' fontWeight={400} color={theme.palette.TwClrTxtSecondary}>
```
Comment (karthikbtf) [inline src/components/Reports/ReportFormAnnual.tsx:398]: A tiny amount of margin top and margin left  (could just be my ocd)?

**Review by karthikbtf — APPROVED** [review]
lgtm!!

File: src/components/Reports/ReportFormAnnual.tsx (hunk 1 above)
Comment (charlie-tf) [reply src/components/Reports/ReportFormAnnual.tsx]: How does 4px look to your eye?

Comment (karthikbtf) [reply src/components/Reports/ReportFormAnnual.tsx]: 4px sounds great!

**Pull request merged**